src/notation/inotationnoteinput.h

```diff
@@ -25,6 +25,7 @@
 #include "async/notification.h"
 #include "notationtypes.h"
 #include "types/retval.h"
+#include "engraving\types\types.h"
```

Comment:
Causes build faliures

Reply:
The correct way would be 
```suggestion
#include "engraving/types/types.h"
```
but it looks like you don't need this #include at all, because you use the typedef from `notationtypes.h`.